<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LevelEditorClass</class>
 <widget class="QMainWindow" name="LevelEditorClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LevelEditor</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTreeWidget" name="treeWidget">
    <property name="geometry">
     <rect>
      <x>1590</x>
      <y>20</y>
      <width>256</width>
      <height>361</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string notr="true">1</string>
     </property>
    </column>
   </widget>
   <widget class="QGroupBox" name="groupBox_Data">
    <property name="geometry">
     <rect>
      <x>1500</x>
      <y>440</y>
      <width>381</width>
      <height>221</height>
     </rect>
    </property>
    <property name="title">
     <string>Data</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1920</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionSaveFile"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionCreate_Static_Entity"/>
   <addaction name="actionCreate_Dynamic_Entity"/>
   <addaction name="actionCreate_Gravity_Area"/>
   <addaction name="actionCreate_Gravity_Well"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionCreate_Static_Entity">
   <property name="text">
    <string>Create Static Entity</string>
   </property>
  </action>
  <action name="actionCreate_Dynamic_Entity">
   <property name="text">
    <string>Create Dynamic Entity</string>
   </property>
  </action>
  <action name="actionSaveFile">
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionCreate_Gravity_Area">
   <property name="text">
    <string>Create Gravity Area</string>
   </property>
  </action>
  <action name="actionCreate_Gravity_Well">
   <property name="text">
    <string>Create Gravity Well</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="leveleditor.qrc"/>
 </resources>
 <connections/>
</ui>
